compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/util/ExpectDeclarationRemover.kt

```diff
@@ -63,7 +63,12 @@ class ExpectDeclarationRemover(val symbolTable: ReferenceSymbolTable, private va
                     return
                 }
 
-                val actualParameter = function.findActualForExpected().valueParameters[index]
+                // If the containing declaration is an `expect class` that matches an `actual typealias`,
+                // the `actual fun` or `actual constructor` for this may be in a different module.
+                // Nothing we can do with those; besides, all default values should already exist, or
+                // the frontend would've complained.
```

Comment:
Correction: turns out, the frontend does *not* complain. Instead, incorrect code is silently generated by the non-IR backend (attempts to call a nonexistent default stub in the imported class) while the IR backend fails with an assertion error in ExpressionCodegen (null value parameter).